lib/config.js

```diff
@@ -16,7 +13,8 @@ function getNodeVersion () {
 }
 
 async function getGitRev () {
-  let cwd = path.resolve(__dirname, "..", "..");
+  let cwd = __filename.indexOf('build/lib/config') !== -1 ?
```

Comment:
agree with this comment, also this logic could probably use a code comment as well (why could it be in one of two places?)